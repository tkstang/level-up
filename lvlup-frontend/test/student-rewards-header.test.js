import Header from '../src/components/Student/student-rewards/header';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('Header should render', () => {
  const reward = shallow(
    <Header />,
  );
  expect(toJson(reward)).toMatchSnapshot();
});
