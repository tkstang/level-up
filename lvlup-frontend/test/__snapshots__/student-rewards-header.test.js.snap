exports[`test Header should render 1`] = `
<h2
  className="header">
  Rewards
</h2>
`;
